### PR TITLE
[Intevation] Correcting rules & cleanup

### PR DESCRIPTION
**Pull request by rugk**
* switched default-on/off (problematic) files as they are now different
* cleaned them up
* activated wald.intevation.org (formerly intevation-problematic.xml)

### CODE REVIEW

**Comment by rugk** [comment]
Fixes https://github.com/EFForg/https-everywhere/issues/2846

File: src/chrome/content/rules/Intevation-GmbH.xml [hunk 1]
```diff
@@ -1,3 +1,10 @@
+<!--
+	Other Intevation rulesets:
+
+		- Greenbone.xml
+		- Intevation-problematic.xml
+		- Wald.Intevation.xml
```
Comment (semenko) [inline src/chrome/content/rules/Intevation-GmbH.xml]: please fix up this comment w/ the new filenames

**Comment by semenko** [comment]
Awesome -- thanks! Also gets rid of a right-sided TLD wildcard -- which is great! :)

**Comment by rugk** [comment]
> Also gets rid of a right-sided TLD wildcard -- which is great! :)

Yes, this had to be changed. :smiley: 
So any more ideas for https://github.com/EFForg/https-everywhere/issues/2846?

**Pull request merged**